visualization/requirements.txt

```diff
@@ -1,8 +1,8 @@
-ansi2html
-dash
-dash_bootstrap_components
-hydra-core
-joblib
-pandas
-pygments
-sshtunnel_requests
+ansi2html==1.9.2
```

Comment:
Let's not fix versions unless there is a good reason to. If we find errors because of package upgrade in the future, we can upper-bound it, but on reason to limit the installed versions otherwise